### PR TITLE
Docs Updates after release 6.5

### LINKED ISSUE #31682: Python In Mantid Exercise Updates
- Some of the exercises here use AlignDetectors, which is now deprecated and should be replaced ApplyDiffCal and ConvertUnits (talk to Diffraction team). 
- Maybe make clear at the top of the solution 1 page that all the data can be found in the TrainingCourseData (from Downloads page)
- Check all solution scripts run! e.g. solution 2B is missing the `from mantid.api import AnalysisDataService as ADS` import

### LINKED ISSUE #34377: Docs Updates
~~https://github.com/mantidproject/mantid/issues/34359#issuecomment-1245442421
https://github.com/mantidproject/mantid/issues/34359#issuecomment-1245498654~~

### PR DESCRIPTION
**Pull request by DanielMurphy22**
**Description of work.**
Add superplot to 06_formatting_plots page
Mention changing layout on gettingstarted/mainwindow basic course 
Add ADS import to 02_pim_sol solution code
Update 03_fit_model_choices to give better fit for EMU data

**To test:**

Build the docs-html target and check that the new links, images and wording are all correct!

Fixes #34377
Fixes #31682

*This does not require release notes* because **minor documentation update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/tutorials/mantid_basic_course/fitting_data/03_fit_model_choices.rst [hunk 1]
```diff
@@ -18,9 +18,9 @@ Multiple peaks + background
 
 It is possible to fit overlapping peaks on a common
 background. Load the file `GEM38370_Focussed` and plot spectrum number 6.
-Zoom in on the 3 peaks around 2.15 Angstroms. Add 3 Gaussian peaks and a Linear Background.
-As can be seen in the screenshot below, the peaks don't have to be of the same type. Run the Fit
-and inspect the results.
+Zoom in on the 3 peaks around 2.15 Angstroms. Add a Linear Background, add a Lorentzian peak
+to the large central peak, and add a Gaussian peak to both of the smaller peaks., as below.
```
Comment (robertapplin) [inline docs/source/tutorials/mantid_basic_course/fitting_data/03_fit_model_choices.rst:22]: I think there is a typo in this line

**Review by robertapplin — CHANGES_REQUESTED** [review]
There are some great additions to the docs here, nice images too :+1: 

I'm happy to approve this, I just think I spotted one typo

File: docs/source/tutorials/mantid_basic_course/fitting_data/03_fit_model_choices.rst (hunk 1 above)
Comment (DanielMurphy22) [reply docs/source/tutorials/mantid_basic_course/fitting_data/03_fit_model_choices.rst:22]: I assume @robertapplin means the `.,` as the typo?

**Comment by DanielMurphy22** [comment]
Really sorry, I have made some more docs changes to also tackle #31682 (see the commit message for details). So @robertapplin you will also need to check the pages I have edited there, including the solutions and exercises edited!

**Review by robertapplin — APPROVED** [review]
Looks good to go, all the scripts run as expected :+1:

**Pull request merged**